iap/make_iap_request.py

```diff
@@ -30,12 +30,16 @@
 OAUTH_TOKEN_URI = 'https://www.googleapis.com/oauth2/v4/token'
 
 
-def make_iap_request(url, client_id):
+def make_iap_request(url, client_id, method='GET', **kwargs):
     """Makes a request to an application protected by Identity-Aware Proxy.
 
     Args:
       url: The Identity-Aware Proxy-protected URL to fetch.
       client_id: The client ID used by Identity-Aware Proxy.
+      method: The request method to use
+              ('GET', 'OPTIONS', 'HEAD', 'POST', 'PUT', 'PATCH', 'DELETE')
+      **kwargs: Any of the parameters defined for the Request class:
+                https://github.com/requests/requests/blob/9616f82905f86173477fdb3bf24f2f7228eed3a0/requests/api.py#L16
```

Comment:
Let's link to the default branch, without a line specified (https://github.com/requests/requests/blob/master/requests/api.py). Our requests version is updated by an automated system, so unless we are very general in our link it will be out of date almost immediately.

Reply:
Done